docs/learn/nexus-rules.md

```diff
@@ -0,0 +1,33 @@
+# Rules for storing data in NeXus
+
+!!! danger "Work in progress"
+
+There are several rules which apply for storing single data items in NeXus. There exists a [summary](https://manual.nexusformat.org/datarules.html) in the NeXus documentation outlining most of these rules. However, this explanation is not exhaustive and thus, we have compiled here additional information.
+
+
+## Namefitting
+
+In general, the names of NeXus group and field items are validated according to these boundaries:
+- Recommended names
+  - lower case words separated by underscores and, if needed, with a trailing number
+
+- Allowed names
+  - any combination of upper and lower case letter, numbers, underscores and periods, except that periods cannot be at the start or end of the string
+  - This statement is equivalent to matching  this regular expression (named `validItemName` in the [nxdl.xsd](https://github.com/nexusformat/definitions/blob/main/nxdl.xsd)) XML Schema file:
+  ```
+  ^[a-zA-Z0-9_]([a-zA-Z0-9_.]*[a-zA-Z0-9_])?$
+  ```
+- Invalid names:
+  - any name not matching this `validItemName` regular expression
+
+Note that this explicitly also means that it is not allowed to have whitestrings " " in NeXus names.
```

Comment:
probably you mean white space, but note that whitespaces include not only space, but also other characters, like tab, new line, etc.

Reply:
Thanks for noticing. I guess none of these are allowed, so it's fine to just change to white space.